package.json

```diff
@@ -17,9 +17,11 @@
   },
   "scripts": {
     "app": "npm run l10n && gulp app",
-    "build": "gulp",
-    "l10n": "pontoon-to-json",
+    "build": "npm run lint && gulp",
     "cleanup": "rm -rf dist && npm run l10n",
+    "l10n": "pontoon-to-json",
+    "lint": "eslint --ext .js --ext .jsx .",
```

Comment:
Then we can eventually add `npm-run-all` and just run all the test:** tasks for `test`.